pages/glossary/background-colors-of-text.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Background Colors Of Text Nodes
+key: background-colors-of-text
+---
+
+All colors of the pixels not part of the [foreground colors](#foreground-colors-of-text) pixels, in an bounding box around each [visible](#visible) character in a [text node](https://dom.spec.whatwg.org/#text), where the width is the [advance width](https://www.w3.org/TR/css-values/#length-advance-measure
+), plus one pixel on the left and right, and the height is the [advance height](https://www.w3.org/TR/css-values/#length-advance-measure).
```

Comment:
can we add a note as to why the additional 1px on left & right and not the other two boundaries?